build.sbt

```diff
@@ -194,28 +194,28 @@ lazy val commonJsLibSettings = gspScalaJsSettings ++ commonLibSettings ++ Seq(
 )
 
 lazy val commonWDS = Seq(
-  version in webpack := "4.43.0",
-  version in startWebpackDevServer := "3.11.0",
+  webpack / version := "4.43.0",
+  startWebpackDevServer / version := "3.11.0",
   webpackConfigFile in fastOptJS := Some(
-    (sourceDirectory in (common, Compile)).value / "webpack" / "dev.webpack.config.js"
+    (common / Compile / sourceDirectory).value / "webpack" / "dev.webpack.config.js"
   ),
   webpackConfigFile in fullOptJS := Some(
-    (sourceDirectory in (common, Compile)).value / "webpack" / "prod.webpack.config.js"
+    (common / Compile / sourceDirectory).value / "webpack" / "prod.webpack.config.js"
   ),
-  webpackMonitoredDirectories += (resourceDirectory in (common, Compile)).value,
-  webpackMonitoredDirectories += ((sourceDirectory in (common, Compile)).value / "webpack"),
-  webpackResources := ((sourceDirectory in (common, Compile)).value / "webpack") * "*.js",
+  webpackMonitoredDirectories += (common / Compile / resourceDirectory).value,
+  webpackMonitoredDirectories += ((common / Compile / sourceDirectory).value / "webpack"),
+  webpackResources := ((common / Compile / sourceDirectory).value / "webpack") * "*.js",
   webpackMonitoredFiles / includeFilter := "*",
   useYarn := true,
-  webpackBundlingMode in fastOptJS := BundlingMode.LibraryOnly(),
-  webpackBundlingMode in fullOptJS := BundlingMode.Application,
+  fastOptJS / webpackBundlingMode := BundlingMode.LibraryOnly(),
+  fullOptJS / webpackBundlingMode := BundlingMode.Application,
   test := {},
-  scalaJSLinkerConfig in (Compile, fastOptJS) ~= { _.withSourceMap(false) },
-  scalaJSLinkerConfig in (Compile, fullOptJS) ~= { _.withSourceMap(false) },
+  Compile / fastOptJS / scalaJSLinkerConfig ~= { _.withSourceMap(false) },
```

Comment:
Does fastOptJS / Compile / scalaJSLinkerConfig work too? If it does, I find it easier to process when glancing over the file quickly.

Reply:
The order is quite specific